File: mobile/src/main/java/treehou/se/habit/core/db/model/controller/VoiceCellDB.java
package treehou.se.habit.core.db.model.controller;

import io.realm.Realm;
import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;
import treehou.se.habit.core.db.model.ItemDB;

public class VoiceCellDB extends RealmObject {

    private String icon;
    private ItemDB item;

    public ItemDB getItem() {
        return item;
    }

    public void setItem(ItemDB item) {
        this.item = item;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public static void save(Realm realm, VoiceCellDB item){
        realm.beginTransaction();
        realm.copyToRealmOrUpdate(item);
        realm.commitTransaction();
    }
}

File: mobile/src/main/java/treehou/se/habit/core/db/model/controller/VoiceCellDB.kt
package treehou.se.habit.core.db.model.controller

import io.realm.Realm
import io.realm.RealmObject
import io.realm.annotations.PrimaryKey
import treehou.se.habit.core.db.model.ItemDB

open class VoiceCellDB : RealmObject() {

    var icon: String? = null
    var item: ItemDB? = null

    companion object {

        fun save(realm: Realm, item: VoiceCellDB) {
            realm.beginTransaction()
            realm.copyToRealmOrUpdate(item)
            realm.commitTransaction()
        }
    }
}
